File: src/leetcode/python/AssignCookies_455.py
class AssignCookies_455(object):
    def findContentChildren(self, g, s):
        """
        :type g: List[int]
        :type s: List[int]
        :rtype: int
        """
        g, s = sorted(g), sorted(s)
        i, j = 0, 0
        res = 0
        for i in range(len(s)):
            if s[i] > g[j]:
                res += 1
                j += 1
            if j == len(s):
                break
        return res

File: src/leetcode/java/AssignCookies_455.java
package leetcode.java;

import java.util.Arrays;

public class AssignCookies_455 {
  public int findContentChildren(int[] g, int[] s) {
    int res = 0;
    Arrays.sort(g);
    Arrays.sort(s);
    int j = 0;
    for (int i = 0; i < s.length; i++) {
      if (s[i] >= g[j]) {
        res++;
        j++;
      }
      if (j >= g.length) break;
    }
    return res;
  }
}
